assets/src/edit-story/components/library/panes/pageTemplates/templateSave.js

```diff
@@ -58,17 +68,30 @@ const SaveButton = styled.button`
       theme.colors.interactiveBg.secondaryHover};
   }
 
-  ${focusStyle};
-`;
+  ${({ isDisabled, theme }) =>
+    isDisabled &&
+    `
+      background-color: ${theme.colors.interactiveBg.disable};
```

Comment:
`disable` color in the Figma file for the button differs from what we currently have and what is in the design system so waiting for confirmation on that one, see [Slack conversation](https://xwp.slack.com/archives/CCCAUAH6F/p1621413464003600?thread_ts=1620728539.281100&cid=CCCAUAH6F).